src/AlgebraicGeometry/Schemes/ProjectiveSchemes/Objects/Constructors.jl

```diff
@@ -59,10 +59,27 @@ end
 # Projective space
 ################################################################################
 
-function projective_space(
-    A::CoeffRingType, 
-    var_symb::Vector{Symbol}
-  ) where {CoeffRingType<:Ring}
+@doc raw"""
+    projective_space(A::Ring, var_symb::Vector{Symbol})
+
+Create the (relative) projective space `Proj(A[x₀,…,xₙ])` over `A` 
+where `x₀,…,xₙ` is a list of `Symbol`s for the variable names.  
+
+# Examples
+```jldoctest
+julia> projective_space(QQ, [:x, :PPP, :?])
+Projective space of dimension 2
+  over Rational Field
+
+julia> homogeneous_coordinate_ring(ans)
+Multivariate Polynomial Ring in x, PPP, ? over Rational Field graded by 
+  x -> [1]
+  PPP -> [1]
+  ? -> [1]
+
+```
+"""
+function projective_space(A::Ring, var_symb::Vector{Symbol})
```

Comment:
if you did this
```suggestion
function projective_space(A::Ring, var_symb::Vector{VarName})
```
then you wouldn't need the next helper method.

(not for this PR but for the future)

@mgkurtz this is another function that expects a variable name list; just like `polynomial_ring`, and `free_associative_algebra` and `free_group`, etc.  -- in a dream world we'd handle all of these uniformly, somehow